pages/home.md

```diff
@@ -8,88 +8,162 @@ permalink: /
+          <div class="column is-half-desktop is-offset-one-quarter-desktop">
+          
+            <p class="content has-text-centered is-large">
+              Pods are like secure personal web servers for data.
+              All data in a pod is accessible
+              accesible via the 
+              <a href="https://solid.github.io/specification/">Solid Protocol</a>.
+              When data is stored in someone's 
+              pod, they control who and what can access it.
             </p>
+            
+            <p class="content has-text-centered is-large">
+            Solid is led by the inventor of the Web, 
+            <b><a href="{{site.baseurl}}/team/">Sir Tim Berners-Lee</a></b>, 
+            to help realize his vision 
+            for its future.</p>
+            
           </div>
         </div>
-      </div>
-    </section>
-    <section class="section">
-      <div class="container">
-        <div class="columns">
-          <div class="column is-offset-1 is-5">
-            <h2 class="title">Like peas in a Pod</h2>
-            <p class="content is-large">
-              Solid provides separate places ("<a href="{{site.baseUrl}}/faqs#pod" title="Frequently Asked Questions - what is a Pod?">pods</a>") for you to store your data, a bit like Dropbox and Google Drive and iCloud, but you have the <strong>power to share</strong> any file or folder or contact or event or whatever <strong>with anyone who has a Solid ID</strong>.  You store your data in any place you trust (including your own computer).
-            </p>
-          </div>
-          <div class="column is-hidden-touch is-offset-1 is-4">
-            <figure class="image">
-              <img src="{{site.baseurl}}/assets/img/like-peas-in-a-pod.svg" alt="[]" />
-            </figure>
+        
+    </div>
+  </section>
+  
+  <section id="landing-cards">
+
+    <div class="container">
+    
+      <div class="columns">
+  
+        <div class="column">
+
+          <div class="card">
+            <div class="card-body has-text-centered section">
+              <figure class="image is-64x64">
+                <img
+                  src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/solid/download.svg"
+                  alt="[]"
+                />
+              </figure>
+              <p class="title is-size-4 is-uppercase">
+                <strong>Store anything</strong>
+              </p>
+              <hr/>
+              <p class="content is-size-5">
+                Any kind of data can be stored in a Solid pod, including regular files that you might store in a Google Drive or Dropbox folder, but it is the ability to store Linked Data that makes Solid special.
```

Comment:
Food for thought: while it may not seem that way when surrounded by Solid people, "linked data" and the eternal promise of interoperability and data that's understandable by both humans and computers would be an instant turn-off for me as someone not coming from the RDF world (but having been there when it was overhyped 15 years ago). So maybe we shouldn't put it this front-and-center without speaking more to why it's useful for Solid to build on that and why-it's-different-this-time-no-really.

Reply:
We've been criticized in the last site design for not explaining what Solid actually is. The vast majority of people will not have any past history with or knowledge of linked data. If we explain why this time is different up front, we'll confuse them all.

Reply:
Interesting perspective @Vinnl - but I'd say there is **_nothing_** "different-this-time" at all - i.e. we're still using the same RDF (basically, v1.1 did clean up some of the [crufty stuff](https://www.urbandictionary.com/define.php?term=cruft)) that was around 15 years ago. So in fact we've nothing really to "speak more to" here. What **_has_** changed though (slowly, but increasingly) is people's realization that RDF really might actually be a-bit-better-and-actually-usable-in-practice, at least more so than all the previous attempts at distributed interop (e.g. SOAP, DCOM, XML-RPC, CORBA, etc.). Probably best articulated by Dan Brickley + Libby Miller (this quote is definitely gonna become a major meme :) - https://book.validatingrdf.com/bookHtml005.html). To your original point though, maybe the text here could allude to that quote perhaps (or just repeat the first three short sentences).